ansible/roles/java-base/tasks/main.yml

```diff
@@ -32,20 +32,35 @@
   include_role:
     name: package-upgrade
 
-- name: download java
-  when: os in "ubuntu1404" and arch == "ppc64"
+- name: download java ppc64
+  when: os in ("ubuntu1404", "centos7") and arch == "ppc64"
   get_url:
     url: https://github.com/AdoptOpenJDK/openjdk8-releases/releases/download/jdk8u172-b11/OpenJDK8_ppc64le_Linux_jdk8u172-b11.tar.gz
     dest: /tmp/
 
-- name: unarchive java
+- name: unarchive java ubuntu1404 ppc64
   when: os in "ubuntu1404" and arch == "ppc64"
   unarchive:
     src: /tmp/OpenJDK8_ppc64le_Linux_jdk8u172-b11.tar.gz
     remote_src: yes
     dest: /home/iojs
   tags: java
 
+- name: unarchive java centos7 ppc64
```

Comment:
these two stanzas have the effect of putting java in /usr/bin/java, where the systemd jenkins unit file (and anyone else) would expect it to be.